guides/release/components/helper-functions.md

```diff
@@ -208,9 +208,25 @@ For the full list of available helpers, you can check the [template helpers API
 ### The `get` helper
 
 The [`{{get}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/get?anchor=get)
-helper makes it easy to dynamically send the value of a variable to another
-helper or component. This can be useful if you want to output one of several
-values based on the result of a getter.
+helper makes it easy to dynamically look up a property on an object or an element in an array. The second argument to `{{get}}` can be a string or an integer depending on the object being accessed.
+
+
+accessing properties on an object with string keys:
+```handlebars
+{{get this.someObject "object_key"}}
+```
+accessing properties on an object with numeric keys:
+
+```handlebars
+{{get this.someObjectWithNumericKeys 0}}
+` ` `
+
+accessing elements in an array:
+
+```handlebars
+{{get this.someArray 0}}
+` ` `
+or outputting one of several values based on result of a getter:
```

Comment:
I think @jamescdavis' suggestion was to add a new line around the code blocks. Can we begin each example with "To ..."?

``````suggestion
To access a property on an object with a string key:

```handlebars
{{get this.someObject "object_key"}}
```

To access the first element in an array:

```handlebars
{{get this.someArray 0}}
```

To access a property on an object with a dynamic key:
``````

Note, I removed the example of object with numeric key. I don't think [numeric key is possible](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors#property_names).

Reply:
that sounds good to me!